### PR TITLE
Rubocop autocorrect, bump version to 1.1.5

### PR DESCRIPTION
**Pull request by pkopac**
* just formatting things (`rubocop -a` @ 0.49.1)
* bumping up version to 1.1.5, so we can release #43

### CODE REVIEW

File: lib/chartmogul/metrics/base.rb [hunk 1]
```diff
@@ -40,17 +39,16 @@ def self.mrr_churn_rate(options = {})
     private
 
     def self.preprocess_params(options)
-      [:start_date, :end_date].each do |param_name|
-        options[param_name.to_s.gsub('_', '-')] = options.delete(param_name) if options[param_name]
+      %i[start_date end_date].each do |param_name|
+        options[param_name.to_s.tr('_', '-')] = options.delete(param_name) if options[param_name]
```
Comment (kenzan100) [inline lib/chartmogul/metrics/base.rb]: is `tr` alias of `gsub`? didn't know that

File: lib/chartmogul/utils/hash_snake_caser.rb [hunk 2]
```diff
@@ -39,12 +39,11 @@ def underscore_key(k)
 
       def underscore(string)
         string.gsub(/::/, '/')
-          .gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2')
-          .gsub(/([a-z\d])([A-Z])/, '\1_\2')
-          .tr('-', '_')
-          .downcase
+              .gsub(/([A-Z]+)([A-Z][a-z])/, '\1_\2')
```
Comment (kenzan100) [inline lib/chartmogul/utils/hash_snake_caser.rb]: what's the indentation rule Rubocop is using here?

File: lib/chartmogul/metrics/base.rb (hunk 1 above)
Comment (pkopac) [inline lib/chartmogul/metrics/base.rb]: > Use tr when you want to replace single characters. tr checks single characters (not via a regular expression)

File: lib/chartmogul/utils/hash_snake_caser.rb (hunk 2 above)
Comment (pkopac) [inline lib/chartmogul/utils/hash_snake_caser.rb]: It aligns it on the same column as method call on line 41.
http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/MultilineMethodCallIndentation

**Pull request merged**